python/ray/serve/backends.py

```diff
@@ -26,6 +27,10 @@ def reconfigure(self, *args, **kwargs):
                 # proxy it manually.
                 return self.wrapped.reconfigure(*args, **kwargs)
 
+            @serve.accept_batch
```

Comment:
add a comment saying this just bypass a check and it should works for non-batch method as well?

Reply:
will do!